### ISSUE TITLE
RedPanda-CPP x64 build incorrecly locks files opened from tree or previous tabs

### ISSUE THREAD

**Issue by bitwisexornot** [royqh1979/RedPanda-CPP#70]
Until I switched to the 32 bit build, which does not have this defect, the x64 build will randomly open most files read only after a crash or even just restart of application. Even after subsequent restart of application. There is no mechanism to deal with it. I verified directory and file locks did not exist with windows tools. Had to restart PC. Switched to 32 bit build, no problems and less crashes. This issue is what made me have to switch builds, as if this happens after 6 hours work, and you still have more to do, you are essentially locked out of the project source code files and have to use a separate text editor and RedPanda to compile.

**Comment by royqh1979** [comment]
Red Panda C++ will open a file in readonly mode, if "options" -> "editor" -> "misc" -> "open system header files in readonly" is turned on, and the file is located in system/project's include folder.

**Comment by bitwisexornot** [comment]
I have had this occur to all source files open after a crash and re-open. They will not unlock either toggling that switch, and they are not in that directory. I want to again not while I have only had this happen a couple times (I just started using app recently, love it) in 32bit mode, I could close the second instance, re-open my project that crashed and they were not read-only. However this bug happened to much in x64 build I had to switch. So that may give you something to work with. 5x more at least.
